### PR TITLE
[YUNIKORN-2626] Add flag to helm chart to disable web container

### PR DESCRIPTION
**Pull request by blueBlue0102**
### What is this PR for?

Add flag to helm chart to disable web container.

### What type of PR is it?

* [x] - Feature

### What is the Jira issue?

- <https://issues.apache.org/jira/projects/YUNIKORN/issues/YUNIKORN-2626>

### CODE REVIEW

File: helm-charts/yunikorn/README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ The following table lists the configurable parameters of the YuniKorn chart and
 | `web.resources.limits.memory`                   | Web app memory resource limit                               | `500Mi`                         |
 | `embedAdmissionController`                      | Flag for enabling/disabling the admission controller        | `true`                          |
 | `enableSchedulerPlugin`                         | Flag for enabling/disabling scheduler plugin mode           | `false`                         |
+| `enableWebService`                         | Flag for enabling/disabling web service           | `true`                         |
```
Comment (chia7712) [inline helm-charts/yunikorn/README.md:109]: please reformat this table

File: helm-charts/yunikorn/templates/service.yaml [hunk 2]
```diff
@@ -30,10 +30,12 @@ spec:
       targetPort: http1
       protocol: TCP
       name: yunikorn-core
+    {{- if .Values.enableWebService }}
```
Comment (chia7712) [inline helm-charts/yunikorn/templates/service.yaml:33]: Could you please apply it to ingress also? If there is no web port, we don't need to configure ingress, right?

**Review by chia7712 — COMMENTED** [review]
@blueBlue0102 thanks for patch. two comments are left

File: helm-charts/yunikorn/templates/service.yaml (hunk 2 above)
Comment (blueBlue0102) [reply helm-charts/yunikorn/templates/service.yaml:33]: Okay! I have applied the changes to the ingress.

**Review by chenyulin0719 — APPROVED** [review]
LGTM

**Comment by blueBlue0102** [comment]
@chia7712 
with web:
image
image
image
image

without web:
image
image
image
image

**Comment by chenyulin0719** [comment]
I've alos validated the changes locally.

**Review by chia7712 — APPROVED** [review]
LGTM

BTW, do we need to update site also?

**Comment by blueBlue0102** [comment]
@chia7712 
Thank you for the review!

I've already created a PR for the site update here: <https://github.com/apache/yunikorn-site/pull/436>.

Please let me know if there are any other changes needed.

Thanks again!

**Pull request closed without merging**